WordPressComKit/RequestRouter.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import Alamofire
+
+public enum RequestRouter: URLRequestConvertible {
+    static let baseURLString = "https://public-api.wordpress.com/rest/v1.1/"
```

Comment:
Versioning is per endpoint, so `v1.1` shouldn't be a part of the base URL. It might work for now but it's destined to need a refactor.